test_package/conanfile.py

```diff
@@ -16,5 +16,6 @@ def imports(self):
         self.copy("*.dylib*", dst="bin", src="lib")
 
     def test(self):
-        os.chdir("bin")
-        self.run(".%sexample" % os.sep)
+        if not tools.cross_building(self.settings):
```

Comment:
Can this be `get_env("CONAN_RUN_TESTS", True)` too?

Reply:
It looks like that variable needs to be defined in the profile, which for us it's currently not. The cross_building func deduces it given the target profile and current host machine so I think that is the right choice. Hopefully that will work out of the box for more people. 

I took the liberty of changing to the cross_building check in the main conanfile for the same reasons listed above.

Reply:
Okay, though I'd recommend bring it up on the Conan docs, to see if the suggestion can be changed.